juniper-jti/templates/deployment.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
```

Comment:
Just curious, What is the reason for apps/v1 here and v1 elsewhere?

Reply:
This is a Kubernetes thing. ConfigMap & Service are part of Kubernetes core API, Deployment is part of the `apps` api group which is why it has the apps namespace here. You can find which objects belong to which group in the [Kubernetes API Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#deployment-v1-apps)